cmake/get_cpm.cmake

```diff
@@ -1,4 +1,5 @@
 set(CPM_DOWNLOAD_VERSION 1.0.0-development-version)
+set(CPM_HASH_SUM "EMPTY_HASH_SUM")
```

Comment:
I don't see this being used anywhere, the hash sum check is still missing, right?

Reply:
you're right, added it right now.